akka-stream-testkit/src/main/scala/akka/stream/testkit/StreamTestKit.scala

```diff
@@ -625,6 +625,18 @@ object TestSubscriber {
       self
     }
 
+    /**
+     * Fluent DSL
+     *
+     * Assert that no message is received for the specified time.
+     * Waits for the default period configured as `akka.test.expect-no-message-default`.
+     * That timeout is scaled using the configuration entry "akka.test.timefactor".
+     */
+    def expectNoMessage(): Self = {
+      probe.expectNoMessage(probe.testKitSettings.ExpectNoMessageDefaultTimeout)
```

Comment:
@2m you updated something, but my comment remains:
```
probe.expectNoMessage()
```

or did I misunderstand something?

Reply:
I misunderstood. Updated again.